diceRoll/diceRollUnitTests.js

```diff
@@ -26,15 +25,22 @@ function runTests(runTF) {
 
 function UnitTest(testName, functionToBeTested, expectedOutput) {
   this.report;
-  this.functionToBeTested = testName;
-  this.functionReturned = functionToBeTested;
-  if (this.functionReturned == expectedOutput) {
-    this.report = this.functionToBeTested + '\n\tPassed.\n\tReturned: ' + this.functionReturned;
-    console.log(this.report);
-    return this.report;
-  } else {
-    this.report = this.functionToBeTested + '\n\tFailed.\n\tReturned: ' + this.functionReturned;
-    console.log(this.report);
-    return this.report;
+  this.testName = testName;
+  this.functionToBeTested = functionToBeTested;
+  this.functionReturned = this.functionToBeTested();
+  this.expectedOutput = expectedOutput;
+  if (typeof this.expectedOutput === 'function') this.expectedOutput = this.expectedOutput();
```

Comment:
suggestion: even though you can use if statements without { }, its better to use them to avoid mistakes and improve readability